src/LibraryPage.tsx

```diff
@@ -162,10 +189,32 @@ const LibraryPage: React.FC = () => {
         <TableColumn col="title">Title</TableColumn>
         <TableColumn col="author">Author</TableColumn>
         <TableColumn col="summary">Summary</TableColumn>
-        <Button onClick={() => setModalOpen(true)}>Edit</Button>
+        <BookEditButton
+          onEdit={(b) => {
+            setIsNewBook(false);
+            setModalOpen(true);
+            if (b) {
+              setSelectedBook(b);
+            }
+          }}
+        ></BookEditButton>
       </AutoTable>
     </PageLayout>
   );
 };
-
+interface editBookProps {
```

Comment:
Could you capitalize this? Interfaces kind of have a role like classes so it helps distinguish them from instance variables.